src/schema.rs

```diff
@@ -330,11 +335,47 @@ impl Team {
         Ok(result)
     }
 
+    pub(crate) fn discord_ids(&self, data: &Data) -> Result<Vec<usize>, Error> {
+        Ok(self
+            .members(data)?
+            .iter()
+            .flat_map(|name| data.person(name).map(|p| p.discord_id()))
+            .flatten()
+            .collect())
+    }
+
     pub(crate) fn is_alumni_team(&self) -> bool {
         self.people.include_all_alumni
     }
 }
 
+#[derive(serde_derive::Deserialize, Debug)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub(crate) struct DiscordRole {
+    name: String,
+    role_id: usize,
```

Comment:
Can we avoid specifying the Role ID here? Ideally a newly created team would just add something like:

```toml
[[discord-role]]
name = "team-name"
```

...and the new role would automatically be created.

Reply:
how would you deal with the role hierarchy?  
![image](https://user-images.githubusercontent.com/10949810/109743201-4eade300-7b9e-11eb-9bf5-58d22f6ddf23.png)

would a new team go to the bottom?

Reply:
A possible solution would be to add new roles after the last role managed by the team repo. So, for example, if you have "managed1", "managed2", "unmanaged1", "managed3", "unmanaged2", new roles would be added after "managed3". if no managed roles are present defaulting to the bottom would be acceptable.

Reply:
> Can we avoid specifying the Role ID here? Ideally a newly created team would just add something like:
> 
> ```toml
> [[discord-role]]
> name = "team-name"
> ```
> 
> ...and the new role would automatically be created.

In discord, names of roles are not uniquie.  That is, 2 roles can have the same name and different ids.  Managing roles exclusively by name means we would have to ensure the uniqueness.

Reply:
@sebasmagri can you send me or post here your discord-id? Im not 100% sure if your actually on discord and I want to make sure I have the right user.

Reply:
Right now we practically have uniqueness in the roles. I don't expect we would create multiple roles with the same name.